rules/library.bzl

```diff
@@ -1022,14 +1022,15 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
         _append_headermap_copts(swift_angle_bracket_hmap_name, "-I", additional_objc_copts, additional_swift_copts, additional_cc_copts)
 
     # Note: this line is intentionally disabled
-    if cpp_sources and False:
+    if cpp_sources:
         additional_cc_copts.append("-I.")
-        native.cc_library(
+        native.objc_library(
```

Comment:
This had to be an `objc_library` to avoid an error in `framework_packaging_impl` where it expected all deps would have an `Objc` provider